hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/marker/WriteMarkersFactory.java

```diff
@@ -20,6 +20,7 @@
 
 import org.apache.hudi.common.fs.FSUtils;
 import org.apache.hudi.common.fs.StorageSchemes;
+import org.apache.hudi.common.table.marker.MarkerType;
```

Comment:
guess this import is unused

Reply:
Previously `MarkerType` is under the same package as `WriteMarkersFactory` so there is no import here.  As I changed the package for `MarkerType`, we need the import here.